lib/ex_gram/updates/webhook.ex

```diff
@@ -0,0 +1,68 @@
+defmodule ExGram.Updates.Webhook do
+  @moduledoc """
+  Updates implementation that uses webhook method
+  """
+
+  use GenServer
+  require Logger
+
+  def update(update, token_hash) do
+    token_hash =
+      token_hash
+      |> String.to_atom()
+
+    GenServer.cast(token_hash, {:update, update})
+  end
+
+  def start_link({:bot, pid, :token, token}) do
+    token_hash =
+      token_hash(token)
+      |> String.to_atom()
+
+    GenServer.start_link(__MODULE__, {:ok, pid, token},
+      name: Module.concat(__MODULE__, token_hash)
```

Comment:
Extract the `Module.concat(__MODULE__, token_hash)` to a method, you will need to use it on the `GenServer.cast` too